participants/yitik-kawatra.md

```diff
@@ -1,5 +1,15 @@
-Name: Yitik Kawatra
-GitHub username: yitik-kawatra
-University: GGSIPU
-Year: 2nd
-Branch: ECE
+<h1>Name: Yitik Kawatra</h1>
+<h2>GitHub username:[yitik-kawatra](https://https://github.com/yitik-kawatra/
+</h2>
+<div>
+<h3>
+<i>
+<ul>
+<li>*University: GGSIPU </li>
```

Comment:
Perhaps the asterisk here was a mistake? You can view your file [here](https://github.com/ACM-IGDTUW/Open-Source-Challenge/blob/bc1cc28f55127610883b74824479899efdabb1f0/participants/yitik-kawatra.md) to see what I am talking about.

Reply:
@lisa761 Thank you I will make the necessary changes and submit again.